docker/Dockerfile

```diff
@@ -69,7 +58,4 @@ COPY ./bin/run.sh /usr/local/bin/run
 # Make our cmd script be executable
 RUN chmod +x /usr/local/bin/run
 
-# Set the working directory for the next commands
-WORKDIR /var/www/html
-
-CMD ["/usr/local/bin/run"]
```

Comment:
I almost feel like having `ENTRYPOINT echo "PHP: $PHP_VERSION"` or something along those lines would make it clear what version of PHP I'm running each time.